### DISCUSSION TITLE
Using up & down arrows for zoom control

### DISCUSSION THREAD

**Discussion by JHuberman** [d2phap/ImageGlass#1571] [General]
I just tried Moon b4, and was not able to use the up and down arrows for zoom.  That is important for me, and I hope it will be possible when it is released.
Also it would be very helpful it it would remember the zoom level for each picture when scrolling through a folder and then use those zoom levels in a slideshow.
In any case, thanks for all your hard work in producing this program.  I wish I could help, but my software skills are with fabric, making fabric tension sculptures.

**Comment by d2phap** [comment]
ImageGlass 9.0 beta supports hotkey customization. To set zooming by up/down keys, you can do this:
```ts
// in igconfig.json file
"MenuHotkeys": {
  "MnuZoomIn": ["Up"],
  "MnuZoomOut": ["Down"]
}
```

You can lock the zoom ratio for all images by using "Lock zoom ratio" mode
![image](https://github.com/d2phap/ImageGlass/assets/3154213/187f57b0-d996-465d-bb15-8ee3cb6720a3)
